articles/concepts/index.md

```diff
@@ -13,7 +13,7 @@ no-loc: ['Q#', '$$v']
 
 A host of new computer technologies has emerged within the last few years, and quantum computing is arguably the technology requiring the greatest paradigm shift on the part of developers.  Quantum computers were proposed in the 1980s by [Richard Feynman](https://en.wikipedia.org/wiki/Richard_Feynman) and [Yuri Manin](https://en.wikipedia.org/wiki/Yuri_Manin).  The intuition behind quantum computing stemmed from what was often seen as one of the greatest embarrassments of physics: remarkable scientific progress faced with an inability to model even simple systems. You see, quantum mechanics was developed between 1900 and 1925 and it remains the cornerstone on which chemistry, condensed matter physics, and technologies ranging from computer chips to LED lighting ultimately rests.  Yet despite these successes, even some of the simplest systems seemed to be beyond the human ability to model with quantum mechanics.  This is because simulating systems of even a few dozen interacting particles requires more computing power than any conventional computer can provide over thousands of years!
 
-There are many ways to understand why quantum mechanics is hard to simulate.  Perhaps the simplest is to see that quantum theory can be interpreted as saying that matter, at a quantum level, is simultaneously in a host of different possible configurations (known as *states*) at the same time.  Unlike classical probability theory, these many configurations of the quantum state, which can be potentially observed, may interfere with each other like waves in a tide pool.  This interference prevents the use of statistical sampling to obtain the quantum state configurations.  Rather, we have to track *every possible* configuration a quantum system could be in if we want to understand the quantum evolution.  
+There are many ways to understand why quantum mechanics is hard to simulate.  Perhaps the simplest is to see that quantum theory can be interpreted as saying that matter, at a quantum level, is in a host of different possible configurations (known as *states*) at the same time.  Unlike classical probability theory, these many configurations of the quantum state, which can be potentially observed, may interfere with each other like waves in a tide pool.  This interference prevents the use of statistical sampling to obtain the quantum state configurations.  Rather, we have to track *every possible* configuration a quantum system could be in if we want to understand the quantum evolution.  
```

Comment:
```suggestion
There are many ways to understand why quantum mechanics is hard to simulate.  Perhaps the simplest is to see that quantum theory can be interpreted as saying that matter, at a quantum level, is in a host with different possible configurations (known as *states*).  Unlike classical probability theory, these many configurations of the quantum state, which can be potentially observed, may interfere with each other like waves in a tide pool.  This interference prevents the use of statistical sampling to obtain the quantum state configurations.  Rather, we have to track *every possible* configuration a quantum system could be in if we want to understand the quantum evolution.  
```

Reply:
..."can be interpreted as saying that matter, at a quantum level, is in a host of different possible configurations (known as *states*) at the same time. "

It might be clearer to say ..."can be interpreted as saying that matter, at a quantum level, is in a multitude of possible configurations (known as *states*) at the same time.

Reply:
I think we should also remove "at the same time".

Reply:
Okay - ..."can be interpreted as saying that matter, at a quantum level, is in a multitude of possible configurations, known as *states*."